packaging/installer-linux/installer-debian/src/main/resources/community/debian/neo4j.postinst

```diff
@@ -29,11 +29,21 @@ case "$1" in
                 --ingroup nogroup --disabled-password --shell /bin/bash \
                 $NEO_USER
         fi
-      
+
+        # remove the old conffiles
+        dpkg-maintscript-helper rm_conffile "/etc/init.d/neo4j-service" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/README.txt" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/neo4j-http-logging.xml" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/neo4j-server.properties" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/neo4j.properties" -- "$@"
+
         # directories needed for neo4j
         chown -R $NEO_USER:adm /var/lib/neo4j /var/log/neo4j /etc/neo4j /usr/share/neo4j /var/run/neo4j
         chmod u+rwx /var/lib/neo4j /var/log/neo4j /etc/neo4j /usr/share/neo4j /var/run/neo4j
 
+        # conf directory symlink, needed because config-migrator depends on config living in conf subdir
+        ln -s /etc/neo4j /var/lib/neo4j/conf
```

Comment:
Do you have a plan for making sure this gets removed when it's no longer needed?